src/WebJobs.Extensions.DurableTask/DurabilityProvider.cs

```diff
@@ -38,12 +39,14 @@ public class DurabilityProvider : IOrchestrationService, IOrchestrationServiceCl
         /// <param name="serviceClient">The internal <see cref="IOrchestrationServiceClient"/> that provides functionality
         /// for this classes implementions of <see cref="IOrchestrationServiceClient"/>.</param>
         /// <param name="connectionName">The name of the app setting that stores connection details for the storage provider.</param>
-        public DurabilityProvider(string storageProviderName, IOrchestrationService service, IOrchestrationServiceClient serviceClient, string connectionName)
+        /// <param name="eventSourceName">The name of the event source provider.</param>
+        public DurabilityProvider(string storageProviderName, IOrchestrationService service, IOrchestrationServiceClient serviceClient, string connectionName, string eventSourceName)
```

Comment:
Instead of feeding this in as a constructor parameter, maybe we should just make EventSourceName a virtual property that subclasses can override.